tools/crm_attribute.c

```diff
@@ -410,9 +410,46 @@ command_delete(pcmk__output_t *out, cib_t *cib)
 {
     int rc = pcmk_rc_ok;
 
-    rc = cib__delete_node_attr(out, cib, cib_opts, options.type, options.dest_node,
-                               options.set_type, options.set_name, options.attr_id,
-                               options.attr_name, options.attr_value, NULL);
+    xmlNode *result = NULL;
+    bool use_pattern = options.attr_pattern != NULL;
+
+    /* See the comment in command_query regarding xpath and regular expressions. */
+    if (use_pattern) {
+        rc = cib__get_node_attrs(out, cib, options.type, options.dest_node,
+                                 options.set_type, options.set_name, NULL, NULL,
+                                 NULL, &result);
+
+        if (!xml_has_children(result)) {
+            goto done_deleting;
+        }
```

Comment:
Not actually necessary, `pcmk__xml_first_child()` will return NULL if there aren't any

Reply:
Fixed.